### PR TITLE
Fixes mob delayers at extremely low ticklags

### LINKED ISSUE #4528: High tickrate makes it impossible to move properly
Since fucking with tick rate is the newest epic thing to do that can't be done on test server:

Too high tick rate makes me unable to move less than 2 tiles, meaning it's impossible to turn corners while space walking around station, or to actually reenter station if the airlock is on an odd tile.

### PR DESCRIPTION
**Pull request by clusterfack**
Fixes #4528 
"Arbitrarily high numbers" need to actually be high if you don't want to break my future shit nexis

Literally does nothing but raise an arbitrary cap to move delays imposed by nexis, because low ticklags require high delays.

### CODE REVIEW

**Comment by dylanstrategie** [comment]
Is there any reason to not use INFINITY ?

**Comment by clusterfack** [comment]
If we used infinity we may as well have it not have a max. But to honest I couldn't really think of a particularly good reason to have a max in the first place.

**Comment by dylanstrategie** [comment]
I suppose it's a sanity, but unless a proc was horribly designed or you are using truly insane ticklags, it shouldn't prop up

**Comment by clusterfack** [comment]
Mob movement was already delaying up to about 30 before. A ticklag of about 60fps brings that up above 100 so its possible to be unable to move less than 1 tile at a time.

File: code/modules/mob/delays.dm [hunk 1]
```diff
@@ -34,11 +34,11 @@
 // Constructor args are currently all the same, but placed here for ease of tuning.
 /client // Yep, clients are snowflakes.
 	// Walking speed is 7, as is grab speed.
-	var/datum/delay_controller/move_delayer    = new (1, 100) // /mob/delayNextMove()
+	var/datum/delay_controller/move_delayer    = new (1,10000) // /mob/delayNextMove()
 /mob
-	var/datum/delay_controller/click_delayer   = new (1, 100) // (Handled in Click())
-	var/datum/delay_controller/attack_delayer  = new (1, 100) // delayNextAttack()
-	var/datum/delay_controller/special_delayer = new (1, 100) // delayNextSpecial()
+	var/datum/delay_controller/click_delayer   = new (1,10000) // (Handled in Click())
```
Comment (ComicIronic) [inline code/modules/mob/delays.dm]: Is this some sort of MAGIC NUMBER
YOU KNOW HOW I FEEL ABOUT THOSE

Comment (clusterfack) [inline code/modules/mob/delays.dm]: These numbers are literally arbitrary

Comment (ComicIronic) [inline code/modules/mob/delays.dm]: Yes, but you also had to edit them multiple times.
Should be a define. This is the exact reason to use a define.

**Pull request merged**